dub.sdl

```diff
@@ -0,0 +1,63 @@
+name "dmd"
+description "The DMD compiler"
+authors "Walter Bright"
+copyright "Copyright © 1999-2017, Digital Mars"
+license "BSL-1.0"
+
+targetType "library"
+
+// This is required because otherwise Dub will include all source files in src.
+// Just picked an existing directory which doesn't contain any D source files.
+sourcePaths "docs"
+
+// This is required because Dub needs to have at lease one source file
+sourceFiles "src/ddmd/package.d"
```

Comment:
Is there really no better solution here? Having to put hacks for our official package manager in our official compiler's package description is ... not great.